docs/open_source/release-process.md

```diff
@@ -29,25 +42,48 @@ git checkout master && git pull && git fetch --tags
 
 This will pull the latest tags and `master` commits into your local repository.
 
-### Additional Step for Patch Releases
+<details open>
+  <summary><strong>Additional Step for Minor Releases and Patch Releases</strong></summary>
+
+First, make sure you checkout the correct branch or tag based on what you're releasing:
+
+* If you're releasing the first non-major version ever after a major version (typically x.0.1), checkout `master`.
```

Comment:
Are you asking me to basically make entirely separate copies of the full process for each type of release? I briefly considered that, but figured it would become _ridiculously_ easy for them to end up out of sync by accident in future updates. That's why I opted for the idea of being able to easily collapse/expand the sections necessary for the particular kind of release being tagged, so it keeps the branches together and thus less prone to forgetting to update one of them.

Reply:
I'm referring to only these 4 bullet points so that we don't need scan through each bullet point for minor & patch releases?

If I'm not wrong these steps belong to their respective release step currently:

**Before**

1) (minor & patch)
2) minor
3) patch
4) patch

**After**

minor
1)
2)

patch
1)
3)
4)

WDYT?